app/components/Search/ResourcesRow.jsx

```diff
@@ -35,6 +35,55 @@ class ResourcesRow extends Component {
     });
   }
 
+  getOpenTime(schedule) {
+    let date = new Date();
+    let day = date.getDay();
+    let days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+    let hour = date.getHours();
+    let closingOpeningTimes = {};
+    let currDay = days[day];
+    let result = { open: false };
+
+    schedule.forEach(item => {
+      closingOpeningTimes[item.day.replace(/,/g, '')] = {
+        open: item.opens_at,
+        close: item.closes_at
+      };
+    });
+
+    if(closingOpeningTimes[currDay] && hour < closingOpeningTimes[currDay].close) {
+      return {
+        open:true, 
+        time: timeToString(closingOpeningTimes[currDay].close)
+      };
+    } else {
+      let remainingDays = days.splice(day+1);
+      remainingDays.some(d => {
+        if(closingOpeningTimes[d]) {
+          result = {
+            open: false,
+            time: `${timeToString(closingOpeningTimes[d].open)} ${d}`
+          };
+          return true;
+        }
+      });
+
+      if(result.time) return result;
+
+      days.some(d => {
```

Comment:
The first loop checks to see if the resource is open any days that are remaining in the week. 

If no open days are found for the remainder of the week, I start at the beginning of the week and check to see the soonest day the resource is open.